sql-parser/src/main/java/io/crate/sql/tree/DropIngestRule.java

```diff
@@ -0,0 +1,44 @@
+package io.crate.sql.tree;
+
+
+import com.google.common.base.MoreObjects;
+
+public class DropIngestRule extends Statement {
+
+    private final QualifiedName name;
+
+    public DropIngestRule(QualifiedName name) {
+        this.name = name;
+    }
+
+    public QualifiedName name() {
+        return name;
+    }
+
+    @Override
+    public int hashCode() {
+        return name.hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
afaik these methods are never used (e.g. instances are not used in any list or map). but ok, all implementations currently do, so maybe I missed something...